src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -54,24 +54,38 @@ public PSCredential Credential
         /// Gets and sets the user supplied message providing description about which script/function is 
         /// requesting the PSCredential from the user.
         /// </summary>
-        [Parameter(Mandatory = true, ParameterSetName = messageSet)]
+        [Parameter(Mandatory = false, ParameterSetName = messageSet)]
+        [ValidateNotNullOrEmpty]
         public string Message
         {
             get { return _message; }
             set { _message = value; }
         }
-        private string _message = null;
+        private string _message;
 
         /// <summary>
         /// Gets and sets the user supplied username to be used while creating the PSCredential.
         /// </summary>
         [Parameter(Position = 0, Mandatory = false, ParameterSetName = messageSet)]
+        [ValidateNotNullOrEmpty]
```

Comment:
This is a breaking change.  A script might use something like:
```PowerShell
Get-Credential -User $sser
````
Note the typo, `$user` was intended.  This script would still run and prompt, just without the user filled in already.

With your change, the script would fail with an unexpected error.